letter-sender/src/test/java/SendChristmasLetterTest.java

```diff
@@ -0,0 +1,113 @@
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.TestInstance;
+import org.testcontainers.shaded.com.fasterxml.jackson.databind.JsonNode;
+import org.testcontainers.shaded.com.fasterxml.jackson.databind.ObjectMapper;
+import org.testcontainers.shaded.com.fasterxml.jackson.databind.node.TextNode;
+import software.amazon.awssdk.auth.credentials.AwsBasicCredentials;
+import software.amazon.awssdk.auth.credentials.AwsCredentialsProvider;
+import software.amazon.awssdk.auth.credentials.StaticCredentialsProvider;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.sns.SnsClient;
+import software.amazon.awssdk.services.sns.model.*;
+import software.amazon.awssdk.services.sqs.SqsClient;
+import software.amazon.awssdk.services.sqs.model.*;
+
+import java.io.IOException;
+import java.net.URI;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class SendChristmasLetterTest {
+
+    public static final String DUMMY = "dummy";
+    private static final String LOCALSTACK_URL = "http://localhost:4566";
+    private static final String LOCALSTACK_ARN = "arn:aws:sqs:us-east-1:000000000000";
+    private static final String REGION = "us-east-1";
+
+    private SnsClient snsClient;
+    private SqsClient sqsClient;
+    private String topicArn;
+    private String queueURL;
+
+    @BeforeAll
+    void setup(){
```

Comment:
Most likely, if we would have more complex functionalities, we would need to kind of have the same setup for the other tests as well, so it's worth extracting this into a new class that can be reused.